Codeinwp/optimole-wp
# Add ToC notice

### All Submissions:

 

### Changes proposed in this Pull Request:
Added ToC notice the signup form. Didn't add it to the API form screen as it suggests user has signed up already and is only connecting now.
 
<img width="1303" alt="Screenshot 2023-07-26 at 10 48 57 AM" src="https://github.com/Codeinwp/optimole-wp/assets/2649903/3413c7fc-a346-46b9-b0cd-6e99f68e241d">


Closes https://github.com/Codeinwp/optimole-service/issues/941.

### How to test the changes in this Pull Request:

1. Make sure you're logged out and see the ToC notice below the connecting form in the initials screen.

### Other information:
